src/alfasim_sdk/_internal/hook_specs_gui.py

```diff
@@ -40,7 +40,7 @@ def alfasim_get_data_model_type():
     .. |m_2| image:: /_static/images/marker_2.png
         :scale: 80%
 
-    |m_1| Location to where the models :func:`~alfasim_sdk.models.container_model` or :func:`~alfasim_sdk.models.data_model` will be placed. |br|
+    |m_1| Location to where the models :func:`alfasim_sdk.container_model` or :func:`alfasim_sdk.data_model` will be placed. |br|
```

Comment:
Please, keep the `~` it will add a link with just the name of the structure instead of the entire path.